src/libraries/Microsoft.PowerFx.Core/Public/Types/FormulaType.cs

```diff
@@ -57,6 +61,25 @@ internal FormulaType(DType type)
             _type = type;
         }
 
+        /// <summary>
```

Comment:
Perhaps we could do without this (it would be nice :)). I'll need to check whether this can be avoided at our side.

Reply:
yes, let's remove. 
Exposing the string format for types as part of public surface area is large surface - and we'll want to to replace with a proper maker-facing syntax.